.kitchen.vagrant.yml

```diff
@@ -1,18 +1,27 @@
 ---
 driver:
   name: vagrant
+  driver_config:
+    http_proxy: <%= ENV['http_proxy'] || nil %>
```

Comment:
I'd like to focus on the docker part since I simply copied the vagrant-configuration and did not change it. I'll change it in another PR.